### PR TITLE
Update readme with new commands

### PR DESCRIPTION
**Pull request by xyan264**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Switchtec Gen4 PAX is a variant of Microsemi's Switchtec PCIe switch product whi
 
 By default, an NVMe drive in the PAX fabric cannot be enumerated by the host. Using the bind operation provided by the Switchtec Gen4 PAX switch, such a VF can be bound to and enumerated by the host.
 
-Unlike some other SR-IOV devices such as NICs, NVMe drives need to be properly configured before they can be used. Generally, configuration includes creating an namespace and attaching the namespace to a specific VF (NVMe secondary controller). 
+Unlike some other SR-IOV devices such as NICs, NVMe drives need to be properly configured before they can be used. Generally, configuration includes creating a namespace and attaching the namespace to a specific VF (NVMe secondary controller), before bringing the VF online.
```
Comment (kelvin-cao) [inline README.md:10]: Virtual resource allocation and assignment are also common configurations for VFs.

File: README.md [hunk 2]
```diff
@@ -93,41 +99,61 @@ create-ns: Success, created nsid:6
...
+7. Allocate VQ and VI resources for the controller
+```
+#sudo ./switchtec-nvme virt-mgmt 0x3300@/dev/switchtec0 -c1 -r0 -a8 -n2
+success, Number of Resources allocated:0x2
+#sudo ./switchtec-nvme virt-mgmt 0x3300@/dev/switchtec0 -c1 -r1 -a8 -n2
+success, Number of Resources allocated:0x2
+```
+8. Reset the controller and then bring it online
+```
+#sudo ./switchtec-nvme switchtec vf-reset 0x3301@/dev/switchtec0 -f
```
Comment (kelvin-cao) [inline README.md:126]: We should add some description about how we know there's a 0x3301 which is a VF.

File: README.md (hunk 1 above)
Comment (xyan264) [reply README.md:10]: Updated.

**Comment by xyan264** [comment]
Updated based on the comments above. Please review.

**Review by kelvin-cao — APPROVED** [review]

**Pull request merged**